gems/govuk_tech_docs/GHSA-x2xw-hw8g-6773.yml

```diff
@@ -0,0 +1,35 @@
+---
+gem: govuk_tech_docs
+ghsa: x2xw-hw8g-6773
+url: https://github.com/alphagov/tech-docs-gem/security/advisories/GHSA-x2xw-hw8g-6773
+title: govuk_tech_docs vulnerable to unescaped HTML on search results page
+date: 2023-04-11
+description: |-
+  'Impact
```

Comment:
Single quotes around block text is unnecessary.

Reply:
fixed